APCAppCore/APCAppCore/UI/Onboarding/EmailVerification/APCEmailVerifyViewController.m

```diff
@@ -467,6 +467,26 @@ - (IBAction) resendEmail: (id) __unused sender
         if (error != nil) {
             APCLogError2 (error);
         }
+        
+        dispatch_async(dispatch_get_main_queue(), ^{
+            NSString *localizedMessage;
+            NSString *localizedAlertActionTitle;
+
+            if (error != nil) {
+                localizedMessage = NSLocalizedStringWithDefaultValue(@"Unable to connect to server, please check your connection", 
+                    @"APCAppCore", APCBundle(), @"Unable to connect to server, please check your connection", @"");
+                localizedAlertActionTitle = NSLocalizedStringWithDefaultValue(@"Retry", @"APCAppCore", APCBundle(), @"Retry", @"");
+            } else {
+                localizedMessage = NSLocalizedStringWithDefaultValue(@"Verification email has been resent",
+                    @"APCAppCore", APCBundle(), @"Verification email has been resent", @"");
+                localizedAlertActionTitle = NSLocalizedStringWithDefaultValue(@"OK", @"APCAppCore", APCBundle(), @"OK", @"");
+            }
+            
+            UIAlertController *alertController = [UIAlertController alertControllerWithTitle:@"" message:localizedMessage preferredStyle:UIAlertControllerStyleAlert];
+            [alertController addAction:[UIAlertAction actionWithTitle:localizedAlertActionTitle style:UIAlertActionStyleDefault handler:nil]];
+            
+            [self presentViewController:alertController animated:YES completion:nil];
```

Comment:
weakSelf and also check that there isn't a presentedViewController on top of this and that this is at the top of the nav stack.  Additionally, I seem to recall that this has a spinner which must first be dismissed. Please verify.